pkg/serverless/interval.go

```diff
@@ -55,8 +56,13 @@ func (d *Daemon) InvocationInterval() time.Duration {
 // This function doesn't mind if the flush strategy has been overridden through
 // configuration / environment var, the caller is responsible of that.
 func (d *Daemon) AutoSelectStrategy() flush.Strategy {
+	flushInterval := 10 * time.Second
 	freq := d.InvocationInterval()
 
+	if !d.clientLibReady {
+		return flush.NewPeriodically(flushInterval)
```

Comment:
Is that the one that makes the most sense when the client lib isn't ready?

Reply:
So the use case we are trying to support is when the extension is used without a client library. AtEnd doesn't work if the /flush route is never called. Ideally we would use 'AtStart', but right now I'm just trying to make sure it doesn't timeout the invocation.